src/apps/mock.ts

```diff
@@ -92,7 +92,11 @@ export function mockListAppsResult(
     .map((a) => a.trim())
     .filter(Boolean)
     .map((name, i) => {
-      const dependencies = getDependencies(name);
+      // FIXME LL-5352 until backend fixes the mistake, remove the dependencies for Decred.
+      const dependencies =
+        name === "Decred" || name === "Decred Testnet"
```

Comment:
please put these in a whitelist object to avoid the double code.

Reply:
Sure, I'll see what I can do boss

Reply:
Putting comments allows the reviewer to understand the reasoning for the development, it's all part of the plan 🧠